common/src/main/java/gov/cms/ab2d/common/dto/UserDTO.java

```diff
@@ -0,0 +1,30 @@
+package gov.cms.ab2d.common.dto;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+import java.util.Set;
+
+@Data
+public class UserDTO {
+
+    @NotNull
+    private String username;
+
+    @NotNull
+    private String firstName;
+
+    @NotNull
+    private String lastName;
+
+    @NotNull
+    private String email;
+
+    @NotNull
+    private Boolean enabled;
+
+    @NotNull
+    private Long sponsorId;
```

Comment:
Again, this requires a DB lookup prior to using the API. Can we use Sponsor HPMS ID, since it is readily available in the HPMS report? I guess not, since there were collisions between  HMPS ID for Parent & Child orgs, if I remember right. How about a combination of HPMS ID and Organization Name then?

Reply:
Child and parent hpms was what I used before so that should work.

Reply:
Could you check whether the combination of HPMS ID and Organization Name is unique? I'd like to avoid having to pass parent/child flags to this API.

Reply:
Oh yeah the hpms/org name is unique.